src/test/java/org/opensearch/ad/HistoricalAnalysisRestTestCase.java

```diff
@@ -231,14 +233,16 @@ protected ADTaskProfile waitUntilGetTaskProfile(String detectorId) throws Interr
         return adTaskProfile;
     }
 
-    protected ADTaskProfile waitUntilTaskDone(String detectorId) throws InterruptedException {
+    protected List<Object> waitUntilTaskDone(String detectorId) throws InterruptedException {
```

Comment:
nit: maybe use some Pair/Tuple class or `AbstractMap.SimpleEntry` rather than `List<Object>` here?

Reply:
How about we improve later? It's too tight to fix. Will add some todo.

Reply:
Seems we are ok to merge later / post-1.1? Up to you, so I'm ok either way.

Reply:
I prefer to merge to 1.1, so we can test BWC automatically when infra team trigger CI workflow.